exhibitor-standalone/src/main/java/com/netflix/exhibitor/standalone/ExhibitorCLI.java

```diff
@@ -21,6 +21,7 @@
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
 import com.netflix.exhibitor.core.Exhibitor;
+import com.netflix.exhibitor.core.azure.PropertyBasedAzureCredential;
```

Comment:
If Azure is in a separate module, we'll need some way to inject azure support here.